src/WebJobs.Script.WebHost/ContainerManagement/LinuxContainerInitializationHostService.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Azure.AppService.Proxy.Common.Log;
```

Comment:
Is that where `Utility` class is? seems like an oddly named namespace for that.

Reply:
No, Utility is in Microsoft.Azure.WebJobs.Script. Not sure why this namespace is needed

Reply:
This is for cleaning the sas url before logging . Line 93 sasUri.CleanUrl()

Reply:
Is that logic complex? For the logging, can we just build and clean the URI? I would like to avoid additional dependencies on proxies if possible.

Reply:
Removed this.